from llaim.exception import LLAIMException


class LLAIMAggregatorException(LLAIMException):
    pass
